sbi/neural_nets/flow.py

```diff
@@ -173,26 +173,79 @@ def build_nsf(
     y_numel = embedding_net(batch_y[:1]).numel()
 
     if x_numel == 1:
-        raise NotImplementedError
+
+        class ContextSplineLinearMap(nn.Module):
+            """
+            Defines a linear map from `context` to the spline parameters.
+
+            In 1D, we have no `conditiner` (see Papamakarios 2019 review for notation).
+            Thus, we can't use the resnet as conditioner. Instead, we just make the
```

Comment:
cannot

Reply:
Suggestion: "We cannot use the reset as conditioner to learn each dimension conditioned on the other dimensions (because there is only one). What we do instead is to learn the spline parameters directly."